src/core_landice/Registry.xml

```diff
@@ -1304,6 +1397,74 @@ is the value of that variable from the *previous* time level!
 	       description="generic work array with dimensions of (nCells)"
 	       persistence="scratch"
 	       />
+	  <var name="meanFlowParamAField" type="real" dimensions="nCells" units="none"
```

Comment:
units and descriptions are updated

Reply:
Can you remove "Field" from all the variable names you've added in this section?  'field' and 'var' are different data types in MPAS, so it would be confusing to have vars with "Field" in the name.  Also, it looks like there may be some new variables added in this section that are not used.  I see `thicknessField` was added but is never used.  I have not checked all of them.  Can you delete any that are not actually used in the code?

Reply:
I was following other scratch field naming. Now I change then to "Var". Some scratch fields are left here but commented out, because I changed them to regular variables for output, like nstar. Can we leave this part a bit later after we are happy with the main trunck of the damage code?

Reply:
@tongzhangice , it doesn't make sense to have 'var' in the name either - these should just be the name of the variable.  e.g. 'meanFlowParamA'

I realize the existing usage of scratch fields is a bit of a mess and it's not clear how they ought to be used!